daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -109,6 +110,10 @@ private[lf] object Speedy {
     private[lf] val stakeholders: Set[Party] = signatories union observers;
   }
 
+  private[this] def enforceLimit(toChecked: Int, limit: Int, error: Int => IError.Limit.Error) =
+    if (toChecked > limit)
+      throw SError.SErrorDamlException(IError.Limit(error(limit)))
```

Comment:
should we make the error include the actual value as well? I personally have found that useful in error messages in the past for other limits (e.g. grpc max message sizes).

Reply:
What do you mean by "actual value" ? Do you mean the size of the set or the actual set ? 
Note that for input Contract the number does not help because the exception will be thrown  always when the size of input contracts is 1 more than the limit.

Reply:
I was suggesting to include both `toChecked` and `limit` in the error. You’re right that it doesn’t help for input contracts but for signatories seeing something like "got 1234 signatories but limit is 100" seems useful. Even if we include the actual list of signatories seeing the number that we compared against is handy imho.

Reply:
067df150f7dfc933ed471da32391bf176d5fd7b2